src/modules/functionality.js

```diff
@@ -0,0 +1,141 @@
+  taskCheckboxContainer.type = 'checkbox';
+  const taskInputContainer = document.createElement('input');
+  taskInputContainer.classList.add('text');
+  taskInputContainer.type = 'text';
+  taskInputContainer.value = task.description;
+  taskInputContainer.setAttribute('readonly', 'readonly');
+  taskContentContainer.appendChild(taskCheckboxContainer);
+  taskContentContainer.appendChild(taskInputContainer);
+
+  // function for editing and removing task from the list
+  const taskActionsContainer = document.createElement('div');
+  taskActionsContainer.classList.add('actions');
+  const taskEditButton = document.createElement('button');
+  taskEditButton.classList.add('edit');
+  taskEditButton.innerText = 'Edit';
+  const taskDeleteButton = document.createElement('button');
+  taskDeleteButton.classList.add('delete');
+  taskDeleteButton.innerText = 'Delete';
+  taskActionsContainer.appendChild(taskEditButton);
+  taskActionsContainer.appendChild(taskDeleteButton);
+  taskContainer.appendChild(taskActionsContainer);
+  listContainer.appendChild(taskContainer);
+  input.value = '';
+
+  taskEditButton.addEventListener('click', () => {
+    if (taskEditButton.innerText.toLowerCase() === 'edit') {
+      taskEditButton.innerText = 'Save';
+      taskInputContainer.removeAttribute('readonly');
+      taskInputContainer.focus();
+    } else {
+      taskEditButton.innerText = 'Edit';
+      taskInputContainer.setAttribute('readonly', 'readonly');
+    }
+  });
+
+  // function for removing task from the list
+  taskDeleteButton.addEventListener('click', () => {
+    listContainer.removeChild(taskContainer);
+  });
+
+  // Remove the todo item from the list of todo items in local storage
+  taskDeleteButton.addEventListener('click', () => {
+    const taskIndex = taskStorage.indexOf(task);
+    taskStorage.splice(taskIndex, 1);
+    saveTodo();
+  });
+
+  // delete all tasks from the list
+  deleteAllList();
+
+  // Remove all todo items from the list in local storage
+  removeAllTodo();
+
+  // function for updating items object's value for completed
+  taskCheckboxContainer.addEventListener('change', () => {
+    if (taskCheckboxContainer.checked === true) {
+      task.completed = true;
+    } else {
+      task.completed = false;
+    }
+    saveTodo();
+  });
+
+  // Error message for no task input
+  if (task.description === '') {
+    taskContainer.remove();
+    const error = document.createElement('div');
+    error.classList.add('error');
+    error.innerText = 'Please enter a task';
+    errorMessage.appendChild(error);
+    setTimeout(() => {
+      error.remove();
+    }, 2000);
+  }
+}
+
+function formSubmission() {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    createTask();
+    addEditRemoveTask(taskObject);
+  });
+}
+
+const populateTasks = () => {
+  if (localStorage.getItem('taskStorage') !== null) {
+    getStoredTodos();
+    taskStorage.map((task) => {
+      addEditRemoveTask(task);
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/207963459-6e0eb395-09c7-4066-8d3c-913798943942.PNG)

**[Optional]**

- [ ] When there are todos then your application should not show `No tasks available!!` 🤔 Kindly fix it 👍

Reply:
Issue fixed. Thank you for your reviews